samples/bluetooth/peripheral_mds/README.rst

```diff
@@ -29,37 +29,38 @@ Overview
 
 In this sample, the Memfault SDK is used as a module in the |NCS| to collect core dumps, reboot reasons, metrics, and trace events from devices and send them through a Bluetooth gateway to the Memfault cloud.
 See `Memfault terminology`_ for more details on the various Memfault concepts.
-The sample includes also the BAS functionalities.
+The sample also includes the BAS functionalities.
 
 Metrics
 =======
 
 The sample shows how to capture user-specific metrics.
 It defines the following metrics:
 
-   * ``Button3PressCount`` - The number of **Button 3** presses.
-   * ``BatteryLvl`` - The simulated battery level.
-   * ``Button1TimeMeasure`` - The time measured between two **Button 1** presses.
+* ``Button3PressCount`` - The number of **Button 3** presses.
+* ``BatteryLvl`` - The simulated battery level.
+* ``Button1TimeMeasure`` - The time measured between two **Button 1** presses.
 
 These metrics are defined in the :file:`samples/bluetooth/peripheral_mds/memfault_config/memfault_metrics_heartbeat_config.def` file.
 For more details about the metrics, see `Memfault: Collecting Device Metrics`_.
 
 There are also metrics that are specific to |NCS|.
-The :ref:`mod_memfault` adds these system proprietary metrics.
+:ref:`mod_memfault` adds these system proprietary metrics.
 The following metrics are enabled by default in this sample:
 
-   * Bluetooth metrics, enabled and disabled using the :kconfig:option:`CONFIG_MEMFAULT_NCS_BT_METRICS` option.
+* Bluetooth metrics, enabled and disabled using the :kconfig:option:`CONFIG_MEMFAULT_NCS_BT_METRICS` Kconfig option.
 
-      * ``Ncs_BtConnectionCount`` - Number of Bluetooth connections.
-      * ``Ncs_BtConnectionTime`` - Bluetooth connection time.
-         Amount of time with at least one live Bluetooth connection.
-      * ``Ncs_BtBondCount`` - Number of Bluetooth bonds.
+  * ``Ncs_BtConnectionCount`` - Number of Bluetooth connections.
+  * ``Ncs_BtConnectionTime`` - Bluetooth connection time.
 
-   * Stack usage metrics shows the free stack space in bytes.
-     Configurable by the :kconfig:option:`CONFIG_MEMFAULT_NCS_STACK_METRICS` option.
+      Time with at least one live Bluetooth connection.
+  * ``Ncs_BtBondCount`` - Number of Bluetooth bonds.
```

Comment:
Shouldn't this be aligned with the bullet point?
```suggestion
    Time with at least one live Bluetooth connection.
  * ``Ncs_BtBondCount`` - Number of Bluetooth bonds.
```

Reply:
Yes, thanks for spotting this.